utils/generic.py

```diff
@@ -608,8 +634,8 @@ def get_all_dups(s, dups, samples_duplicates):
     return duplicated_samples
 
 
-def infer_families(kin_ht: hl.Table,
-                   sex: Dict[str, bool],
+def infer_families(relatedness_ht: hl.Table,
+                   sex: Union[hl.Table, Dict[str, bool]],
```

Comment:
when i was going through this with a cmg cohort i had to add the following otherwise i was sometimes getting siblings output as the parents:
"ibd1_parent_offspring_threshold: float = 0.75" and then check that ibd1 was above this value and added in code as necessary downstream

Reply:
Can you PR this change? Would be great!